client/js/analysisui/layoutcell.js

```diff
@@ -65,9 +72,9 @@ var LayoutCell = Backbone.View.extend({
 
         if (this.$previousContent !== null)
             this.trigger('layoutcell.contentChanged', Math.random());
```

Comment:
Yep will do. It is basically a content change identifier so that grids know whether or not they have handled the event already as it propagates up the tree and then back down. So they don't do it twice.